src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -20,7 +20,12 @@ case class RocketCrossingParams(
   mmioBaseAddressPrefixWhere: TLBusWrapperLocation = CBUS
 ) extends TileCrossingParamsLike {
   def injectClockNode(context: Attachable)(implicit p: Parameters): ClockNode = {
-    ClockTempNode() // TODO reset stretcher could go here, parameterized by args to this case class
+    if (p(SubsystemResetSchemeKey) != ResetSynchronous) {
```

Comment:
Rather than implicitly coupling usage of this class to `SubsystemResetSchemeKey` specifically, can we make `stretchReset: Boolean = false` or `stretchResetCycles: Option[Int] = None` be one of the case class parameters above, and then set that accordingly in places where we instantiate a `RocketCrossingParams`, e.g.
```
case XCrossingKey => RocketCrossingParams(..., stretchReset = site(SubsystemResetSchemeKey) != ResetSynchronous))
``` (there seem to be only a couple relevant usages in closed source)

Reply:
Changed to Option[Int] parameter.